src/components/Cocktail.jsx

```diff
@@ -1,4 +1,18 @@
 import React from 'react';
 
+function Cocktail() {
+  return (
+    <div className="Cocktail">
+      <div>
+        <img/>
+      </div>
+      <div>
+        {const toto= }
```

Comment:
is this line valid?

Reply:
there is a mistake, we will fix it soon.